Applied conventions on testing/1.7.0

madebr opened the pull request:
Hello,

`bincrafters-conventions` and `conan-readme-generator` were executed on the branch testing/1.7.0

madebr committed:
Run conan-readme-generator

commit by conan-repo-actions

madebr committed:
Run bincrafters-conventions

commit by conan-repo-actions

madebr committed:
Add author, topics, license and sha256

Croydon commented on conanfile.py:
```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     homepage = 'http://www.openh264.org/'
     description = "Open Source H.264 Codec"
-    license = "BSD 2-Clause"
+    topics = "conan", "h264", "codec", "video", "compression",
```
once again:

please use braces in the future `topics = (`

madebr replied:
ok, is this checked in bincrafters-conventions?

Croydon commented on conanfile.py:
```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     name = "openh264"
     version = "1.7.0"
     url = "https://github.com/bincrafters/conan-openh264"
+    author = "bincrafters <bincrafters@gmail.com>"
```
uppercase `Bincrafters`

madebr replied:
ok, is this checked in bincrafters-conventions?

madebr committed:
Fix topics and author

madebr committed:
Fix sha256

I had downloaded two v1.7.0.tar.gz archives from 2 different projects and did not notice that this has been saved as v1.7.0.tar.gz.2.
Oops!
